modules/ROOT/pages/document-types.adoc

```diff
@@ -27,7 +27,6 @@ Valid values:
 Default:
 None
 
-
 |Version - X12 only
 
 |Select a version from the drop-down menu: `3010`, `3020`, `3030`, `3040`, `3050`, `3060`, `3070`, `4010`, `4020`, `4030`, `4040`, `4050`, `4060`, `5010`, `5020`, `5030`, `5040`, `5050`, `6020`
```

Comment:
Do we really need to list all of these here? We did not for the previous row, which used a similar ref. to the drop-down menu.... Also, be sure to use "and" or "or" at the completion of such a list:
```suggestion
|Select a version from the drop-down menu: `3010`, `3020`, `3030`, `3040`, `3050`, `3060`, `3070`, `4010`, `4020`, `4030`, `4040`, `4050`, `4060`, `5010`, `5020`, `5030`, `5040`, `5050`, or `6020`.
```

Reply:
Line 39
Select a value from the drop-down list, or enter a different value.

Reply:
Line 51
Auto-populates > Automatically populates a value

Reply:
Line 60-62
|Clicking *Use Standard* accepts the standard you named and selected; clicking *Custom* enables you to choose a file to download.
You must use a custom schema for JSON. Supported schema types are .schema, .JSON, and .ESL (for X12).